pallets/capacity/src/helpers_tests.rs

```diff
@@ -221,14 +221,13 @@ fn set_capacity_details_is_successful() {
 
 		assert_eq!(Capacity::get_capacity_for(target), None);
 
-		let mut capacity_details = CapacityDetails::<
-			BalanceOf<Test>,
-			<Test as frame_system::Config>::BlockNumber,
-		>::default();
-
-		capacity_details.remaining = 10;
-		capacity_details.total_available = 10;
-		capacity_details.last_replenished_epoch = 1;
+		let capacity_details: CapacityDetails<BalanceOf<Test>, <Test as Config>::EpochNumber> =
```

Comment:
... EpochNumber is 0-indexed, but here the CapacityDetails value is explicitly set.